storage/reads/store.go

```diff
@@ -84,3 +85,18 @@ type Store interface {
 
 	GetSource(orgID, bucketID uint64) proto.Message
 }
+
+// WindowAggregateCapability describes what is supported by WindowAggregateStore.
+type WindowAggregateCapability interface {
+	query.WindowAggregateCapability
+}
+
+// WindowAggregateStore implements the WindowAggregate capability.
+type WindowAggregateStore interface {
```

Comment:
I think we do not need to make a separate WindowAggregateStore for OSS. All those designs were for rolling upgrade in idpe.